src/change_stream.ts

```diff
@@ -416,9 +430,34 @@ export class ChangeStreamCursor extends Cursor<AggregateOperation, ChangeStreamC
     }
   }
 
+  close(): void;
+  close(callback: Callback): void;
+  close(options: CursorCloseOptions): Promise<void>;
+  close(options: CursorCloseOptions, callback: Callback): void;
+  close(
+    optionsOrCallback?: CursorCloseOptions | Callback,
+    callback?: Callback
+  ): Promise<void> | void {
+    this.emitter.emit(Cursor.CLOSE);
```

Comment:
We haven't completely settled on a pattern for this, but I think the majority of our code still does something like:
```
close(options?: CursorCloseOptions | Callback<void>, callback?: Callback<void>): Promise<void> | void {
  if (typeof options === 'function') (callback = options), (options = {});
  options = options || {};
  ...
}
```
It types just fine, let's you use simple names for the parameters, and doesn't require you to do things like casting `callback` below